intdataplane/int_dataplane.go

```diff
@@ -591,6 +606,31 @@ func (d *InternalDataplane) apply() {
 		}
 		d.cleanupPending = false
 	}
+
+	// Set up any needed rescheduling kick.  Any pending kick will have been recalculated above
+	// so we always cancel and reschedule.
```

Comment:
I don't understand what the last sentence here is trying to tell me.  What would be the alternative to cancelling and rescheduling, and why does choosing or not choosing that depend on having calculated the reschedDelay above?